.travis.yml

```diff
@@ -16,22 +16,24 @@ matrix:
       sudo: required
       env:
         - ARCH=x86_64
+        - CGO_ENABLED=0
       go: 1.11.4
       script:
         - make
         - diff -au <(gofmt -s -d cmd) <(printf "")
         - diff -au <(gofmt -s -d pkg) <(printf "")
-        - for d in $(go list ./... | grep -v browser); do go test -v -race --timeout 15m "$d"; done
+        - for d in $(go list ./... | grep -v browser); do CGO_ENABLED=1 go test -v -race --timeout 15m "$d"; done
```

Comment:
Why is it needed to selectively turn on CGO?

Reply:
Race requires cgo in Go 1.11